src/bindings/jna/libelektra/src/main/java/org/libelektra/KDBException.java

```diff
@@ -0,0 +1,335 @@
+package org.libelektra;
+
+import static org.libelektra.ValidationUtil.argNotNull;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import javax.annotation.Nonnull;
+import org.libelektra.exception.ConflictingStateException;
+import org.libelektra.exception.InstallationException;
+import org.libelektra.exception.InterfaceException;
+import org.libelektra.exception.InternalException;
+import org.libelektra.exception.KeyReleasedException;
+import org.libelektra.exception.OutOfMemoryException;
+import org.libelektra.exception.PluginMisbehaviorException;
+import org.libelektra.exception.ResourceException;
+import org.libelektra.exception.SemanticValidationException;
+import org.libelektra.exception.SyntacticValidationException;
+
+/**
+ * Wraps Elektra errors into the corresponding Java exceptions
+ *
+ * @see #releaseErrorKey()
+ */
+public abstract class KDBException extends Exception
+{
+
+	protected static final String META_KEY_NOT_FOUND_VALUE = "(unknown)";
+	protected static final String MSG_UNKNOWN_ERRROR_NUMBER =
```

Comment:
Is this MSG_ prefix needed? (And typo)

```suggestion
	protected static final String UNKNOWN_ERROR_NUMBER =
```

Reply:
I think it helps to differentiate format string message constants from other constants to be used in another context. I would leave it for now, but if you like to remove it, just say a word.
type fixed.